### PR TITLE
Add secrets detection check for env variable

### PR DESCRIPTION
**Pull request by YuliiaKovalova**
Add secrets detection check for env variable and target for the template that copies deps.

### CODE REVIEW

File: dotutils-buildchecks.sln
Comment (JanKrivanek) [inline dotutils-buildchecks.sln:1]: Just curious - why to remove this?

File: src/UsedEnvironmentVariablesCheck.cs [hunk 1]
```diff
@@ -0,0 +1,113 @@
+﻿using DotUtils.MsBuild.SensitiveDataDetector;
+using Microsoft.Build.Experimental.BuildCheck;
+using Microsoft.Build.SensitiveDataDetector;
+using Microsoft.Build.Shared;
+using System;
+using System.Collections.Generic;
+
+namespace DotUtils.BuildChecks
+{
+    public sealed class UsedEnvironmentVariablesCheck : Check
```
Comment (JanKrivanek) [inline src/UsedEnvironmentVariablesCheck.cs]: The name is a bit missleading

**Review by JanKrivanek — APPROVED** [review]
Looks great!

File: src/Directory.Packages.props [hunk 2]
```diff
@@ -5,5 +5,6 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageVersion Include="Microsoft.Build" Version="17.13.0-dev-24518-01" />
+    <PackageVersion Include="DotUtils.MsBuild.BinlogRedactor.SensitiveDataDetector" Version="0.0.9" />
```
Comment (JanKrivanek) [inline src/Directory.Packages.props:9]: Btw. this feels as a quite a version bump (0.0.4-beta -> 0.0.9)

Comment (YuliiaKovalova) [reply src/Directory.Packages.props:9]: It's a package reference
https://www.nuget.org/packages/DotUtils.MsBuild.BinlogRedactor.SensitiveDataDetector/

Comment (JanKrivanek) [reply src/Directory.Packages.props:9]: 🤦  :-D
Sry - I was just too quick going through this

**Pull request merged**